### ISSUE TITLE
Automatically close block comment on /

### ISSUE THREAD

**Issue by sharwell** [dotnet/roslyn#30089]
I was encouraged by work from @Neme12 in #30074, so I'm resurrecting a *very* old feature request of mine. 😄 

**Steps to Reproduce**:

Press <kbd>/</kbd> in the location marked with `|`:

```csharp
/*
 * Typed some text and then pressed enter...
 * |
```

**Expected Behavior**:

```csharp
/*
 * Typed some text and then pressed enter...
 */
```

**Actual Behavior**:

```csharp
/*
 * Typed some text and then pressed enter...
 * /
```

**Comment by Neme12** [comment]
Can you please write down the exact specification of when this feature should trigger?

For example, would the comment be closed if I type `/` here?
```c#
/* Some text... * |
```

Basically, what is the exact condition that has to be true in order for this feature to apply? Does there simply have to be a space after an asterisk in a multi-line comment? Does the text on that line until the asterisk have to be all whitespace? Does it have to be exactly 1 space? Is there a check that there's an asterisk on each line above this asterisk? Is there an additional check that there's exactly 1 space before that asterisk on each line? Any combination of these?

**Comment by sharwell** [comment]
1. The complete like text is an indented asterisk followed by a single space

**Comment by sharwell** [comment]
:memo: Today I tried to find my original feature request. The design is included in a repository converted from P4 to Subversion on 10 January 2010, but I don't have a record before that from my original feature proposal. Needless to say, I have been waiting a long, long time for this. 👍
